test/runtests.jl

```diff
@@ -286,15 +286,19 @@ end
                 @test isa(graph_nodes[1], Op)
                 @test graph_nodes[1].func == split
                 @test collect(graph_nodes[1].args) == ["foo bar"]
-                @test collect(graph_nodes[1].kwargs) == [(:limit, 1)]
+                if VERSION < v"0.7"
```

Comment:
Best practice is to find out exactly when this changed, but I don't care enough to demand that